src/server.h

```diff
@@ -1158,39 +1158,48 @@ typedef struct {
 typedef struct client {
     uint64_t id;            /* Client incremental unique ID. */
     uint64_t flags;         /* Client flags: CLIENT_* macros. */
+    listNode clients_pending_write_node; /* list node in clients_pending_write list.
+                                            accessed together with flags field */
     connection *conn;
+    sds querybuf;           /* Buffer we use to accumulate client queries. */
+    size_t qb_pos;          /* The position we have read in querybuf. */
+    int reqtype;            /* Request protocol type: PROTO_REQ_* */
```

Comment:
`reqtype`, `resp` and `authenticated` can be `uint_8`
`slot` can be `uint_16`.
if we rearrange these variables, we also can consider to change their types as well to reduce memory usage